README.md

```diff
@@ -5,52 +5,68 @@
 
 ## Quickstart
 
-Before you can begin with Yari, you need [Content](https://github.com/mdn/content).
-See its README which basically, says something like this:
+Before you can start working with Yari, you need a copy of the MDN [Content](https://github.com/mdn/content)
+directory — this is the content Yari will render when we run it locally. The
+easiest way to get a local copy of this is to run the following command
+(although bear in mind that you might want to clone your own fork of the repo
+instead, for when you create pull requests against the content):
 
     git clone https://github.com/mdn/content.git mdn/content
 
-Now, you just need to note where that folder is before you can start Yari.
+Take a note of where you've clone that repo before moving on.
 
 To run Yari locally, you'll first need to install [git](https://git-scm.com/),
-[Node.js](https://nodejs.org) (>= 12.0.0) and
+[Node.js](https://nodejs.org) (>= 12.0.0), and
 [Yarn 1](https://classic.yarnpkg.com/en/docs/install).
-After that, run these commands in your bash:
+
+After that, run these commands in your terminal, bearing in mind that you'll
+probably want to fork the repo first and do the `git clone` with _your_ fork
+instead of the `mdn` one:
 
     git clone https://github.com/mdn/yari.git
     cd yari
-    yarn
+
+Next run the `yarn` command to fetch all the dependencies and install the app
+locally:
```

Comment:
Remove this sentence and just include `yarn install` as a 3rd command in the code block above. People will be impatient here and look for blocks of commands they can just copy and paste. If this third instruction is nestled into the prose sentence, it might get missed.

Reply:
Done. I agree here. Also `yarn install` is more intuitive in terms of what the command is actually doing, for those not familiar with yarn specifically.